src/viking.js

```diff
@@ -1,11 +1,119 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health=health
+        this.strength=strength
+    }
+
+    attack() {
+        return this.strength
+    }
+
+    receiveDamage (damage){
+    this.health -= damage;
```

Comment:
Try keep the same indentation through your code :) 
```suggestion
         this.health -= damage;
```